tests/context.py

```diff
@@ -276,37 +284,41 @@ def checkCovCommand(args=[None]):  # skipcq: PYL-W0102  - [] != [None]
 
 		Testcase 1: Function should return unmodified arguments if 'coverage' is missing.
 
-			>>> _context.checkCovCommand(["python", "script.py"])
+			>>> _context.checkCovCommand("python", "script.py")
 			['python', 'script.py']
 
 		Testcase 2: Function should modify arguments when 'coverage' is the first argument.
 			A.) Missing 'run'
 
-			>>> _context.checkCovCommand(["coverage", "script.py"])  #doctest: +ELLIPSIS
+			>>> _context.checkCovCommand("coverage", "script.py")  #doctest: +ELLIPSIS
 			['...', 'run', '-p', '--context=Integration', '--source=multicast', 'script.py']
 
 		Testcase 3: Function should modify arguments when 'coverage run' is in the first argument.
 			A.) NOT missing 'run'
 
-			>>> _context.checkCovCommand(["coverage run", "script.py"])  #doctest: +ELLIPSIS
+			>>> _context.checkCovCommand("coverage run", "script.py")  #doctest: +ELLIPSIS
 			['...', 'run', '-p', '--context=Integration', '--source=multicast', 'script.py']
 
 		Testcase 4: Function should handle coverage command with full path.
 
-			>>> _context.checkCovCommand(["/usr/bin/coverage", "test.py"])  #doctest: +ELLIPSIS
+			>>> _context.checkCovCommand("/usr/bin/coverage", "test.py")  #doctest: +ELLIPSIS
 			['...', 'run', '-p', '--context=Integration', '--source=multicast', 'test.py']
 
 		Testcase 5: Function should handle coverage invoked via sys.executable.
 
 			>>> import sys as _sys
 			>>> test_fixture = [str("{} -m coverage run").format(_sys.executable), "test.py"]
-			>>> _context.checkCovCommand(test_fixture)  #doctest: +ELLIPSIS
+			>>> _context.checkCovCommand(*test_fixture)  #doctest: +ELLIPSIS
 			[..., '-m', 'coverage', 'run', '-p', '...', '--source=multicast', 'test.py']
 
 
 	"""
 	if sys.__name__ is None:  # pragma: no branch
-		raise ImportError("[CWE-758] Failed to import system. WTF?!!")
+		raise ImportError("[CWE-758] Failed to import system.") from None
+	if not args or args[0] is None:
+		raise ValueError("[CWE-1286] args must be an array of positional arguments") from None
+	else:
+		args = [*args]  # convert to an array
```

Comment:
This would need to be tested on a seperate development branch, it is a little odd here, but it causes no real harm, this is fine for now, perhaps in the future as part of #134 it may be removed.